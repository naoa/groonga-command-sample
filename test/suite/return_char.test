register commands/echo

echo "hello world\n"
