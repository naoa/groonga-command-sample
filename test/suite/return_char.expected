register commands/echo
[[0,0.0,0.0],true]
echo "hello world\n"
[[0,0.0,0.0],["hello world\n",12]]
